modules/ROOT/pages/_partials/symbolicate-jvm.adoc

```diff
@@ -0,0 +1,17 @@
+// BEGIN -- inclusion -- {root-partials}symbolicate-jvm.adoc
```

Comment:
I don't think this section is applicable for Java. Java / Android will have their own way for symbolicating the symbols. Please check with @bmeike.

Reply:
Turns out that this file didn't have a menu item, so I think it was deleted on purpose. I've removed the file.